diesel_tests/tests/update.rs

```diff
@@ -234,6 +234,28 @@ fn update_with_no_changes() {
         .unwrap();
 }
 
+#[test]
```

Comment:
Since this test is just copied from the one above, it might make sense to add the new line just above the panic one above. Or just keep it as its own test, up to you.

Reply:
It's better to have a separate test for this. These tests are quite fast so there is no point to worry about anything.